src/steamship/agents/schema/llm.py

```diff
@@ -15,3 +16,15 @@ class LLM(BaseModel, ABC):
     def complete(self, prompt: str, stop: Optional[str] = None) -> List[Block]:
         """Completes the provided prompt, stopping when the stop sequeunce is found."""
         pass
+
+
+# TODO(dougreid): should LLM and ConversationalLLM share a common parent?
+class ConversationalLLM(BaseModel, ABC):
+    """ConversationalLLM wraps large language model-based backends that use a chat completion style interation.
+
+    They may be used with LLMAgents in Action selection, or for direct prompt completion."""
```

Comment:
This comment seems to imply that ConversationalLLM should inherit from LLM though

Reply:
probably needs to be updated.